rich/pretty.py

```diff
@@ -524,6 +543,28 @@ def __str__(self) -> str:
             )
 
 
+def _is_namedtuple(obj: Any) -> bool:
+    """Checks if an object is most likely a namedtuple. It is possible
+    to craft an object that passes this check and isn't a namedtuple, but
+    there is only a minuscule chance of this happening unintentionally.
+
+    Args:
+        obj (Any): The object to test
+
+    Returns:
+        bool: True if the object is a namedtuple. False otherwise.
+    """
+    base_classes = getattr(type(obj), "__bases__", [])
+    if len(base_classes) != 1 or base_classes[0] != tuple:
+        return False
+
+    fields = getattr(obj, "_fields", None)
+    if not fields or not isinstance(fields, tuple):
```

Comment:
`_fields` *could* be an empty tuple here. For the strange edge-case of a named tuple with no fieds.

Perhaps just return `isinstance(fields, tuple)` ?